src/main/java/org/folio/roles/domain/model/CapabilityReplacements.java

```diff
@@ -0,0 +1,11 @@
+package org.folio.roles.domain.model;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+public record CapabilityReplacements(Map<String, Set<String>> oldCapabilitiesToNewCapabilities,
+                                     Map<String, Set<UUID>> oldCapabilityRoleAssignments,
```

Comment:
format

Reply:
I'll change it here manually, but this is actually how our formatter config reformats records.
And this is a new code, not an existing one - so it's not that I changed existing formatting. FYI.